.flake8

```diff
@@ -0,0 +1,12 @@
+[flake8]
+select =
+    E
+    W
+    F
+ignore =
+    W503 # makes Flake8 work like black
+    W504
+    E203 # makes Flake8 work like black
+    E741
+max-line-length = 99
```

Comment:
In dbt-core we removed the max-line-length and added E501 to the ignore, because of test contents. Black will format the code correctly to 99.